lib/driver.js

```diff
@@ -416,6 +426,8 @@ export class XCUITestDriver extends BaseDriver {
         this.validateDesiredCaps({...caps, ...this.cliArgs});
       }
 
+      this.printDeprecatedCaps();
```

Comment:
it also makes sense to mark the capability definition in desired-caps as `@deprecated`

Reply:
with `useSimpleBuildTest`

```
[XCUITestDriver@7d57]   ]
[XCUITestDriver@7d57] }
[BaseDriver] useSimpleBuildTest is a deprecated capability
[BaseDriver] useSimpleBuildTest is a deprecated capability
[XCUITestDriver@7d57] The desired capabilities include neither an app nor a bundleId. WebDriverAgent will be started without the default app
[XCUITestDriver@7d57 (0d9a20c1)] Session created with session id: 0d9a20c1-6379-4a0e-8033-2238f4bba6d2
```


without `useSimpleBuildTest`

```
[XCUITestDriver@8e46] }
[BaseDriver] useSimpleBuildTest is a deprecated capability
[BaseDriver] useSimpleBuildTest is a deprecated capability
[BaseDriver] useSimpleBuildTest is a deprecated capability
[XCUITestDriver@8e46] The desired capabilities include neither an app nor a bundleId. WebDriverAgent will be started without the default app
[XCUITestDriver@8e46 (555c4c93)] Session created with session id: 555c4c93-b2d3-47b1-aaf4-a24783cb9ccc
```

interesting.. maybe some behavior in the base driver has wrong handling

Reply:
yes, it might be a bug...
Could you please check it when you have time?